models/login_source.go

```diff
@@ -632,6 +632,9 @@ func UserSignIn(username, password string) (*User, error) {
 		switch user.LoginType {
 		case LoginNoType, LoginPlain, LoginOAuth2:
 			if user.ValidatePassword(password) {
+				if user.ProhibitLogin {
```

Comment:
Also should check in `ExternalUserLogin`

Reply:
@lunny thanks! I've force pushed a change

Reply:
@romankl and could you also add check `IsActive` I think maybe it's missed here.